documentation/src/main/wiki/AdvancedSetup.md

```diff
@@ -0,0 +1,77 @@
+### Advanced Setup
+
+This section describes advanced setup of AET using Linux and Windows. 
+
+Please note that full list of required tools and its versions can be found in [System Components](SystemComponents) section.
```

Comment:
I'm not sure this notation:
`[System Components](SystemComponents)`
will work in wiki. For sure this will:
`[[System Components|SystemComponents]]`

Reply:
Fixed